pkg/variantregistry/ocp_test.go

```diff
@@ -223,6 +223,36 @@ func TestVariantSyncer(t *testing.T) {
 				VariantLayeredProduct:   VariantNoValue,
 			},
 		},
+		{
+			job: "periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-sno-serial",
+			variantsFile: map[string]string{
+				"Topology": "single", // should be ignored
```

Comment:
I don't think this applies as the expected Topology is in fact 'single'.  Unfortunately it looks like the test above is incorrect as well and the likely source.  Could you remove the variantesFile in both places just confirm it isn't picking that up.  Thanks!

Reply:
Updated.  The e2e passed.